tests/helper_test.py

```diff
@@ -0,0 +1,235 @@
+# encoding=utf-8
+# Copyright © 2016 Dylan Baker
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test suite for alot.helper module."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
I can drop them.

__future__ imports are not like other imports though, they don't actually import code, they're directives to the interpreter that just happen to use the `import` syntax, but don't add code, they change how the interpret treats certain language constructs.